charts/prometheus-statsd-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: prometheus-statsd-exporter
 description: A Helm chart for prometheus stats-exporter
-version: 0.1.0
+version: 0.1.1
```

Comment:
new feature should raise minor version to 0.2.0

Reply:
Bumped to `0.2.0`